api/models.js

```diff
@@ -159,7 +159,17 @@ var projectSchema = mongoose.Schema({
     },
 
     quota: {type: Number, default: 1000000000000}, //maximum archive size (1TB by default)
-
+    mag: {
+        papers: { type: [{  
+                publicationDate : {type : Date}, 
```

Comment:
You don't need "type" here because all you are doing is just setting to Date. You can do publicationDate: Date,